packages/npm-dependencies-validation/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    "rootDir": ".",
```

Comment:
Keeping these settings consistent with other sub-packages
in this monorepo until we can investigate and align everything to one standard.